V2/tests/api/azure/customer_azure_endpoints_test.py

```diff
@@ -47,3 +47,117 @@ def test__customer_azure_endpoint__returns_a_status_400__when_dont_recieve_all_n
 
     assert response.status_code == 400
     assert response.get_body() == b'Invalid format or structure of the attributes of the customer'
+
+
+def test__delete_customer_azure_endpoint__returns_a_customer_with_true_deleted__when_its_id_is_found(
+    test_db, customer_factory, insert_customer
+):
+    customer_preinsert = customer_factory()
+    inserted_customer = insert_customer(customer_preinsert, test_db).__dict__
+
+    req = func.HttpRequest(
+        method='DELETE',
+        body=None,
+        url=CUSTOMER_URL,
+        route_params={"id": inserted_customer["id"]},
+    )
+
+    response = azure_customers._delete_customer.delete_customer(req)
+    customer_json_data = json.loads(response.get_body().decode("utf-8"))
+
+    assert response.status_code == 200
+    assert customer_json_data['deleted'] is True
+
+
+def test__update_customer_azure_endpoint__returns_an_updated_customer__when_customer_has_all_necesary_attributes(
+    test_db, customer_factory, insert_customer
+):
+    existent_customer = customer_factory()
+    inserted_customer = insert_customer(existent_customer, test_db).__dict__
+
```

Comment:
this part is repeated in all tests could it be a fixture?
 ````
existent_customer = customer_factory()
inserted_customer = insert_customer(existent_customer, test_db).__dict__
````

and the variable `existent_customer ` is not clear, it could be `customer_preinsert` o `customer_to_insert`